sample_challenge.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+PROMPT_COMMAND="$(pwd)/go-term-adventure $(pwd)/sample_challenge.gta" HISTFILE=$HOME/.advhistory bash
```

Comment:
@Foxiy I see. That will have to wait for another comment which will transform this into some sort of a template, that will take care of most of these things and only require you to specify the file that specifies the respective levels.

It short: it's outside of the scope of this PR.